src/js/modal-download.js

```diff
@@ -158,6 +158,11 @@ function createElement (item) {
 		item.embargoMessage = item.embargoPeriod ? interpolate(MESSAGES.EMBARGO, item) : '';
 	}
 
+	const hasTranslationComponent = document.getElementById('ftlabsTranslationContainer');
```

Comment:
is this the best way to check, could this possibly be based on a flag instead (if they are/can be available)

Reply:
I'm not sure how to get flags in `n-syndication`. Also, even if the flag is on, the translations component might not be shown (there's a whole bunch of conditions handled in `next-article` so I thought it best to check the component on the page. I could combine with the flag if you let me know how.